src/app/pages/LiquidityMining/components/RemoveLiquidityDialog/RemoveLiquidityDialogV1.tsx

```diff
@@ -181,7 +181,7 @@ export const RemoveLiquidityDialogV1: React.FC<IRemoveLiquidityDialogV1Props> =
                 translations.common.availableBalance,
               )} ${weiToNumberFormat(balance, 8)}`}
               maxAmount={balance}
-              dataActionId="yieldFarm"
+              dataActionId="yieldFarm-withdraw"
```

Comment:
```suggestion
              dataActionId="yieldFarm-withdraw-confirmButton-{{ASSET}}"
```

Please update this to include the asset name as per spec (similar to change in `CurrencyRow.tsx`).

Same thing (except with `deposit` in id), for the Add Liquidity modal button in `AddLiquidityDialogV1.tsx`

Reply:
{{asset}} already included here - src/app/components/Form/AmountInput/index.tsx (80 line)